src/json/to_json.cr

```diff
@@ -227,20 +227,9 @@ module Enum::ValueConverter(T)
   # used for serialization.
   #
   # ```
-  # enum Stages
```

Comment:
I had removed it because the test failed with the definition of the enum of the same name.
However, the clarity of the example should be a priority, so I reverted and will modify my tool. 👍